src/client/commands/wait.rs

```diff
@@ -0,0 +1,68 @@
+use crate::JobId;
+
+use crate::rpc_call;
+use clap::Clap;
+
+use crate::client::globalsettings::GlobalSettings;
+use crate::client::status::is_terminated;
+use crate::server::bootstrap::get_client_connection;
+use crate::transfer::messages::{
+    FromClientMessage, JobInfo, JobInfoRequest, JobSelector, ToClientMessage,
+};
+use std::thread::sleep;
+use std::time::Duration;
+
+#[derive(Clap, Copy, Clone)]
+#[clap(setting = clap::AppSettings::ColoredHelp)]
+pub struct WaitOpts {
+    job_id: JobId,
```

Comment:
I suppose that we can leverage job id range parsing and wait for a range of jobs?